rust/cubesqlplanner/cubesqlplanner/src/planner/sql_evaluator/sql_nodes/auto_prefix.rs

```diff
@@ -5,6 +5,8 @@ use crate::planner::sql_evaluator::MemberSymbol;
 use crate::planner::sql_evaluator::SqlEvaluatorVisitor;
 use crate::planner::sql_templates::PlanSqlTemplates;
 use cubenativeutils::CubeError;
+use lazy_static::lazy_static;
```

Comment:
Why not use smth like [OnceCell ](https://doc.rust-lang.org/std/cell/struct.OnceCell.html) or [OnceLock](https://doc.rust-lang.org/std/sync/struct.OnceLock.html) (if thread-safe is needed) from the std library rather than depend on an external crate?

Reply:
Because to do that, I’d need to rewrite them across the whole Tesseract. I’ll do it, but as a separate task. For now, it’s better to use lazy_static! everywhere than end up with a zoo.